pages/pipelines/notifications.md

```diff
@@ -27,6 +27,33 @@ Available notification types:
 * [Webhooks](#webhooks): Send a notification to the specified webhook URL.
 * [PagerDuty](#pagerduty-change-events)
 
+<table>
+<thead>
+  <tr><th>Step-level</th><th>Build-level</th></tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Slack</td>
+    <td>Slack</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Email</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Basecamp</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Webhook</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>PagerDuty</td>
+  </tr>
+</table>
+
```

Comment:
Thanks for this PR @ivannalisetska !

For aesthetic reasons, I'd suggest switching the columns of the table around from the largest to the more granular concept (i.e. build then step), and add an introduction to this table.

```suggestion
These types of notifications are available at the following levels.

<table>
<thead>
  <tr><th>Build</th><th>Step</th></tr>
</thead>
<tbody>
  <tr>
    <td>Slack</td>
    <td>Slack</td>
  </tr>
  <tr>
    <td>Email</td>
    <td></td>
  </tr>
  <tr>
    <td>Basecamp</td>
    <td></td>
  </tr>
  <tr>
    <td>Webhook</td>
    <td></td>
  </tr>
  <tr>
    <td>PagerDuty</td>
    <td></td>
  </tr>
</table>

```